landing-page/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+  <head>
+    <style>
+      body {
+  background-color: MistyRose;
+}
+      </style>
+  </head>
+  <body>
+      <style>
+          h1{
+  background-color: white;
+  color: MistyRose;
+  font: 60pt Impact, Charcoal, sans-serif;
+  display: flex;
+  justify-content: center;
+}
+h2{
+  display: flex;
```

Comment:
this is a small thing but it's more conventional to have a space between the selector and the opening brace.